### PR TITLE
I am not sure why this was not here before

### PR DESCRIPTION
**Pull request by fespinoza**

### CODE REVIEW

File: public/quotes.js [hunk 1]
```diff
@@ -154,5 +154,9 @@ var quotes = [
   {
     quote: "We have developers who kind of develop",
     author: "Andreas"
+  },
+  {
+    quote: "...(typing)...",
+    author: "Andreas"
   }
```
Comment (mathiasbno) [inline public/quotes.js:161]: Is it possible to add a `,` in the end here so in the future we only se the object added not an extra line in the top?

Comment (fespinoza) [reply public/quotes.js:161]: i think that is not valid JSON

Comment (mathiasbno) [reply public/quotes.js:161]: Silly JSON!

Comment (onmyway133) [reply public/quotes.js:161]: Can it be in italic, since it's not quite a saying

Comment (sindrenm) [reply public/quotes.js:161]: @mathiasbno: Yeah, that would be nice!

@fespinoza: This is not a JSON file. :wink: 

@onmyway133: Agree, that would probably be nice, but we don't have support for formatting the quotes now. Would need to add support for something like Markdown or perhaps even just write it in HTML in here for that.

**Pull request merged**